src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/RunWorkflowTests.kt

```diff
@@ -74,8 +74,7 @@ class RunWorkflowTests : SeleniumTest()
         }
     }
 
-    @ParameterizedTest
-    @EnumSource(ConfigType::class)
```

Comment:
realised these only work if you also pass the config type as an argument to the tets, but updating all these tests to be properly parameterized felt like a bigger job